### PR TITLE
Project 15 - Caroline Fahlgren

### PR DESCRIPTION
**Pull request by CarolineF3**
Link to my deployed project: https://todo-carolinef.netlify.app/

### CODE REVIEW

File: code/src/components/ToDoList.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          />
+          <div className='todo-text-container'>
+            <label htmlFor={task.id}>{task.description}</label>
+          </div>
+          <img
+            src={wastebin}
+            alt='Remove'
+            className='waste-bin'
+            onClick={() => dispatch(tasks.actions.removeTask(task.id))}
+          ></img>
```
Comment (Andrealarsson) [inline code/src/components/ToDoList.js]: I can't see your wastebin on the webpage to remove each item? 🤔

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,36 @@
...
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <Provider store={store}>
+      <div className='wrapper'>
+        <img
+          src={require("./images/floral-background.png")}
+          alt='floral-background'
+        />
```
Comment (Andrealarsson) [inline code/src/App.js]: Really cute image 💮

File: code/src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  return (
+    <div className='header'>
+      <h1>To do</h1>
+      <div className='date-number-of-tasks-container'>
+        <p>
+          {new Date().toLocaleDateString("en-us", {
+            month: "long",
+            day: "numeric",
+          })}
```
Comment (Andrealarsson) [inline code/src/components/Header.js:17]: This is a great solution to show the date 👍

**Review by Andrealarsson — APPROVED** [review]
Great work on this project! 💯 Your todo app looks very cute and clean. The code is easy to read and understand. You are awesome Caroline! 🤩 🙌

File: code/src/components/ToDoList.js (hunk 1 above)
Comment (KarolCislowski) [reply code/src/components/ToDoList.js]: Yes this is the only thing that is missing to fulfill the blue requirements 😭

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
It is a very stylish app, I love the flower details and the font face. 
You need to add the button to remove a single task from the list to meet the requirements 😃

**Review by KarolCislowski — APPROVED** [review]
Great App!

**Pull request closed without merging**